File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1146.lua
while(true) do
    n = io.read("*number")
    res = ""
    if(n==0) then
        break
    end
    for i=1,n-1,1 do
        res=res..i.." "
    end
    res=res..n
    print(res)
end

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1146.cs
using System;

class URI
{
    static void Main() {
        int n;
        while(true){
            n = int.Parse(System.Console.ReadLine().Trim());
            string res = "";
            if(n==0){
                break;
            }
            for(int i=1;i<n;i++){
                res+=i+" ";
            }
            res+=n;
            Console.Write("{0}\n", res);
        }
        
    }
}